src/cow-react/common/pure/CurrencyInputPanel/CurrencyInputPanel.tsx

```diff
@@ -137,7 +138,7 @@ export function CurrencyInputPanel(props: CurrencyInputPanelProps) {
           <div>
             {balance && !disabled && (
               <>
-                <styledEl.BalanceText title={balance.toExact() + ' ' + currency?.symbol}>
+                <styledEl.BalanceText title={balance.toExact() + ' ' + formatSymbol(currency?.symbol)}>
```

Comment:
Are we sure we want to trim token symbols in the title?
It's hidden and wouldn't impact the UI. Seeing the whole thing there would be beneficial IMO.

Reply:
Actually we shouldn't and this was an oversight.